common/commands/src/validator/account/send_multiple.rs

```diff
@@ -0,0 +1,234 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use chrono::{DateTime, Utc};
```

Comment:
afaik `time` is preferred to `chrono` (and we use it everywhere over our codebase)